src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -1842,6 +1842,9 @@ protected static CSharpCompilation CreateCompilationWithSpan(SyntaxTree tree, CS
             return comp;
         }
 
+        protected static CSharpCompilation CreateCompilationWithSpan(string s, CSharpCompilationOptions options = null)
```

Comment:
I suspect this change was included accidentally.

Reply:
Nope, the tests only had a helper taking SyntaxTree, so I added one that takes string for convenience.